### PR TITLE
FEAT: show message when an user use a legacy BCH address

### PR DESCRIPTION
**Pull request by rastajpa**
bitpay_-_secure_bitcoin_wallet

### CODE REVIEW

**Comment by Gamboster** [comment]
LGFM. Great idea!

File: src/components/info-sheet/info-sheet.html [hunk 1]
```diff
@@ -112,5 +112,15 @@
...
     </info-sheet-template>
+
+    <info-sheet-template *ngSwitchCase="'legacy-address-info'" type="info">
+      <span sheet-title-text translate>{{params.appName}} BCH wallets use the CashAddr format by default</span>
+      <span sheet-text translate>
+        If you need to send to "old" addresses, and you are SURE those are BCH addresses, you can "translate" them using the BCH
+        Translator tool.
+      </span>
+      <span sheet-button-text translate>Translate address</span>
+      <span sheet-second-button-text translate>GOT IT</span>
```
Comment (cmgustavo) [inline src/components/info-sheet/info-sheet.html]: This action should clear the input, isn't it?

Comment (rastajpa) [reply src/components/info-sheet/info-sheet.html]: If you think that it's better for user experience, I will do it now!

Comment (rastajpa) [reply src/components/info-sheet/info-sheet.html]: Done!

**Pull request merged**